browser/src/Services/Commands.ts

```diff
@@ -14,10 +14,13 @@ import * as UI from "./../UI/index"
 
 import { CallbackCommand, CommandManager } from "./CommandManager"
 
+import * as Platform from "./../Platform"
 import { replaceAll } from "./../Utility"
 
 export const registerBuiltInCommands = (commandManager: CommandManager, pluginManager: PluginManager, neovimInstance: INeovimInstance) => {
     const config = Config.instance()
+    commandManager.clearCommands()
```

Comment:
Thanks for going the extra mile here to wire up showing add / remove correctly, that's awesome!